Update license requirements. (#112)
GEM-benchmark/NL-Augmenter: docs/doc.md

**Novelty:** Your transformation must improve the coverage of NL-Augmenter in a meaningful way. The idea behind your transformation need not be novel, but its contribution to the library must be different from the contributions of earlier submissions. If you are unsure if your idea would constitute a new contribution, please email the organizers at nl-augmenter@googlegroups.com and we are happy to help.

> **Adding New Libraries:** We welcome addition of libraries which are light and can be installed via `pip`. Every library should specify the version number associated and be added in a new [requirements.txt](../transformations/punctuation) in the transformation's own folder. However, we discourage the use of heavy libraries for a few lines of code which could be manually written instead. Please ensure that all the libraries have either Apache 2, MIT License **or a similar license** like BSD. For a different license like BSD, you would need to add the appropriate license file into your transformation/filter folder too. If you are unsure, please email the organizers at nl-augmenter@googlegroups.com. 

Reviewer: ```suggestion
**Adding New Libraries:** We welcome addition of libraries which are light and can be installed via `pip`. Every library should specify the version number associated and be added in a new [requirements.txt](../transformations/punctuation) in the transformation's own folder. However, we discourage the use of heavy libraries for a few lines of code which could be manually written instead. Please ensure that all libraries have MIT, Apache 2, BSD, or other permissive license. GPL-licensed libraries are not approved for NL-Augmenter. If you are unsure, please email the organizers at nl-augmenter@googlegroups.com. 
```

I don't think it makes sense for only the person that added a pip dependency to include the license in their directory, when later submitters may also use that library. I also think it's too much overhead for everyone to look up the licenses of all the libraries they use. So I got rid of the adding-license-file-to-your-directory requirement.

I might add another point to the review criteria:
```
**Data and code source:** The `README.md` file should have a subsection titled "Data and code provenance", which should describe where data or code came from, or that it was fully created by the task author. This section should also disclose the license that any external data or code is released under.
```
